src/test/run-pass/abi-sysv64-arg-passing.rs

```diff
@@ -98,7 +98,7 @@ mod tests {
     #[derive(Copy, Clone)]
     pub struct Floats { a: f64, b: u8, c: f64 }
 
-    #[link(name = "rust_test_helpers")]
+    #[link(name = "rust_test_helpers", kind = "static")]
```

Comment:
Yeah unfortunately no crater coverage just yet, but to be clear the failure mode here was:

* The linkage was incorrectly tagged
* The library *only* pulled in statics
* We then spit out a linker error 

If that's the case then yeah seems like ok breakage. There's a way to fix it to work on all rustc versions, and otherwise we're just fixing a bug.